### ISSUE TITLE
no option to switch facebook account

### ISSUE THREAD

**Issue by sajanyamaha** [roughike/flutter_facebook_login#281]
dont have an option to switch facebook account , even if you logout next time login asks "You previously logged in  xxx with facebook , would you like to continie ?" How can we use another account to login , irony is facebook has provided different test users to try on but how to switch accounts ?

![image](https://user-images.githubusercontent.com/6435237/85204484-d829c780-b325-11ea-9393-fd22ff357a4b.png)

**Comment by RaashVision** [comment]
True, I stuck on this too. You can switch the account if you manually clear the browser cache. But that's not user friendly at all.

**Comment by RaashVision** [comment]
Bro change the flutter login  from
**facebookLogin.loginBehavior = FacebookLoginBehavior.nativeWithFallback;**
to
**facebookLogin.loginBehavior =FacebookLoginBehavior.webViewOnly;**

If you change fb behaviour to webview, it will render thr fb login page everytime you logout from facebook plugin.

**Comment by vishwesh-aubergine** [comment]
@RaashVision  For WebViewOnly I still get this problem

**Comment by RaashVision** [comment]
I realize its only works in android not ios. On ios no option to change the account.

**Comment by vishwesh-aubergine** [comment]
Can you give gist for android ? @RaashVision

**Comment by RaashVision** [comment]
The one I mention above is what I tried bro. I tried in Android and IOS. Android works, IOS doesn't. Some say this plugin is obsolete. Try check out this linke,
https://medium.com/@karlwhiteprivate/flutter-facebook-sign-in-with-firebase-in-2020-66556a8c3586

**Comment by KerollosMelad** [comment]
I have the same issue, 
I want to login with a different account but I cannot, any help ?!

**Comment by RaashVision** [comment]
I have change to the solution belo. It's more reliable. Have tested with both android and ios I testflight. Working just fine.
 https://medium.com/@karlwhiteprivate/flutter-facebook-sign-in-with-firebase-in-2020-66556a8c3586

**Comment by RaashVision** [comment]
Try  flutter_facebook_auth. It's quite stable compared to this plugin. They keep updating this plugin too. Highly recommended.

**Comment by albrownwood** [comment]
same issue
